values.yaml

```diff
@@ -212,8 +212,21 @@ st2web:
       cpu: "50m"
     limits:
       memory: "100Mi"
-  # TODO: Add Ingress and/or LoadBalancer setting as a way to expose service to public (#6)
-  # ingress:
+  # TODO: Add Ingress setting as a way to expose service to public (#6).
+  ingress: {}
+  #  hostname:
```

Comment:
This looks confusing because we're not adding yet an `Ingress` controller as K8s resource.
Especially keeping in mind `service.hostname`.